apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -102,6 +104,49 @@ const ScheduleDirection = ({
           shapeId={shapeId}
         />
       )}
+      {lineDiagram &&
+        lineDiagram.map(
+          ({ route_stop: routeStop, stop_data: stopData }: LineDiagramStop) => (
+            <div className="m-schedule-line-diagram__stop">
+              <div className="m-schedule-line-diagram__card-left">
+                <div className="m-schedule-line-diagram__stop-name">
+                  {routeStop.name}
+                </div>
+                <div>{stopData.type === "terminus" && stopData.branch}</div>
+                <div className="m-schedule-line-diagram__connections">
+                  {routeStop.connections.map((route: Route) =>
+                    route.type === 3 && !route.name.startsWith("SL") ? (
```

Comment:
@meagonqz let's extract these lines into a helper method

Reply:
I'd like to hold off on that since this is scaffolding and this part needs additional UI work anyway (like handling multiple commuter rail icons). It will change so I don't see the need to move it into a separate file yet.